src/components/Layout/Header/components/ConnectButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button } from 'components';
+import { useState } from 'react';
+import { UnlockPanel, UnlockButton, ProviderFactory } from 'lib';
+import { ExtendedProviders } from 'initConfig';
+import { IProviderFactory } from '@multiversx/sdk-dapp-core/out/core/providers/types/providerFactory.types';
+import { useNavigate } from 'react-router-dom';
+import { RouteNamesEnum } from 'localConstants';
+
+export const ConnectButton = () => {
+  const [open, setOpen] = useState(false);
+  const navigate = useNavigate();
+
+  const handleLogin = async ({ type, anchor }: IProviderFactory) => {
+    const provider = await ProviderFactory.create({
+      type,
+      anchor
+    });
+    await provider?.login();
+    navigate(RouteNamesEnum.dashboard);
+  };
+
+  return (
+    <>
+      <Button onClick={() => setOpen(true)}>Connect</Button>
+      <UnlockPanel
+        open={open}
+        onLogin={(options) =>
+          handleLogin({
+            type: options.detail.provider,
+            anchor: options.detail.anchor
+          })
+        }
+        onClose={() => {
+          setOpen(false);
+        }}
+      >
+        <UnlockButton
```

Comment:
add a flag on top of the file with this option hidden by default
This is an advanced option and it's not for every day use
Also add the comment: Advanced login optoin. It can be safely removed if not needed